app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java

```diff
@@ -894,21 +1014,21 @@ public boolean onFling(MotionEvent e1, MotionEvent e2, float velocityX, float ve
             final float absVelocityX = Math.abs(velocityX);
             final float absVelocityY = Math.abs(velocityY);
             if (absVelocityX > shutdownFlingVelocity) {
-                onClose();
+                closePopup();
```

Comment:
Since now we have a proper quick close option, it'd be a good time to remove this swipe to close (shutdown fling) feature.

Reply:
Removed.